src/MintNFT.js

```diff
@@ -0,0 +1,59 @@
+import "./MintNFT.css";
+import nftPlaceholder from "./images/nftPlaceHolder.svg";
+import { useSDK } from "@metamask/sdk-react";
+import { ethers } from "ethers";
+import { useEffect, useState } from "react";
+import MyNFT from "./artifacts/contracts/TicketNFT.sol/TicketNFT.json";
+const MintNFT = () => {
+  const nftId = "9527";
+  const ethProvider = new ethers.providers.Web3Provider(window.ethereum);
+  const contract = new ethers.Contract(
+    "0xfF13D2d139f1997B66DF4D97A1A6A127A615A9eB",
+    MyNFT.abi,
+    ethProvider
+  );
```

Comment:
config related code in above lines (and also other lines/files) will be moved to `.env` soon, i put this here right now to prevent merge conflicts